src/layer/tile/GridLayer.js

```diff
@@ -102,11 +102,11 @@ L.GridLayer = L.Layer.extend({
 		bounds: null,
 
 		// @option minZoom: Number = 0
-		// The minimum zoom level that tiles will be loaded at. By default the entire map.
+		// The minimum zoom level that tiles will be loaded at. By default the entire map.  This will override the [minimum zoom level](#map-minzoom) set on the map.
```

Comment:
Would be better with something like «If the map's [`minZoom`](#map-minzoom) option is set to a lower value than this, then this layer will not be displayed when zooming fully out.»